fe/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts start",
+    "dev": "react-scripts start",
```

Comment:
Please also keep the `run` script as it's really a standard with react and many people are used to starting their apps with it.

Reply:
but there wasn't a run script

Reply:
Ah sorry, I meant the `start` script

Reply:
well usually it is used `start` -> starts dev server `dev` -> builds dev assets once doesn't watch and no server. If you are used to `dev` and I understand as in some places `dev` is used same way as `start` then you can either add both to be same and add comment explaining it or keep it as just `start`. I don't see any problems keeping both as cra is not using `dev` command anyway. I suggest to code the script like this `"dev": "npm run start"` or yarn variant if you are using yarn. It would then be copy of `start` without need to change it if `start` would get extra arguments.